### PR TITLE
Change domain name to publish to

### PR DESCRIPTION
**Pull request by Thorinwasher**
# Description
I saw that we have not changed the domain name to publish Mockbukkit to, this pr should fix that.

Domain name is now `org.mockbukkit.mockbukkit` instead of `com.github.seeseemelk`

# Checklist
The following items should be checked before the pull request can be merged.
- [n/a] Code follows existing style.
- [n/a] Unit tests added (if applicable).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,10 +7,10 @@
         <img alt="Documentation Status" src="https://readthedocs.org/projects/mockbukkit/badge/?version=v1.21" />
     </a>
     <a href="https://search.maven.org/search?q=MockBukkit">
-        <img alt="Maven Central" src="https://img.shields.io/maven-central/v/com.github.seeseemelk/MockBukkit-v1.21?color=1bcc94&logo=apache-maven" />
+        <img alt="Maven Central" src="https://img.shields.io/maven-central/v/org.mockbukkit.mockbukkit/mockbukkit-v1.21?color=1bcc94&logo=apache-maven" />
     </a>
     <a href="https://javadoc.io/doc/com.github.seeseemelk/MockBukkit-v1.21">
-        <img alt="Javadocs" src="https://javadoc.io/badge2/com.github.seeseemelk/MockBukkit-v1.21/javadoc.svg" />
+        <img alt="Javadocs" src="https://javadoc.io/badge2/com.github.seeseemelk/mockbukkit-v1.21/javadoc.svg" />
```
Comment (4everTheOne) [inline README.md]: Will the javadocs be uploaded to "com.github.seesemelk"?

File: README.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ MockBukkit can easily be included in your project using either Maven or gradle.
 > [!TIP]
 > Currently, the newest version available is
 >
-> [![ALTERNATE-TEXT](https://img.shields.io/maven-central/v/com.github.seeseemelk/MockBukkit-v1.21?color=1bcc94&logo=apache-maven)](https://search.maven.org/search?q=MockBukkit)
+> [![ALTERNATE-TEXT](https://img.shields.io/maven-central/v/com.github.seeseemelk/mockbukkit-v1.21?color=1bcc94&logo=apache-maven)](https://search.maven.org/search?q=MockBukkit)
```
Comment (4everTheOne) [inline README.md]: Will this also keep the com.github.seesemelk?

Comment (Thorinwasher) [reply README.md]: Ahh, good catch!

File: README.md (hunk 1 above)
Comment (Thorinwasher) [reply README.md]: Yes, should definitely change

**Review by 4everTheOne — APPROVED** [review]

**Pull request merged**